domains/jamir-garro.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Personal Portfolio",
+    "owner": {
+        "username": "Doki2007",
+        "email": "jamirgarroarana@hotmail.com",
+        "discord": "jamir4538"
+    },
+    "record": {
+        "CNAME": "doki2007.github.io/portfolio/"
```

```suggestion
        "CNAME": "doki2007.github.io"
```
CNAMEs cannot contain slashes.